src/core.js

```diff
@@ -820,6 +823,7 @@ goog.require('olcs.core.OlLayerPrimitive');
    */
   olcs.core.olPointGeometryToCesium = function(geometry, projection, style,
       billboards, opt_newBillboardCallback) {
+    var altitudeMode = geometry.get('altitudeMode');
```

Comment:
Oh, I was not aware that the KML change we discussed got merged. In this case it makes sense I think. So please merge after adding some documentation about the new feature to olcs.VectorSynchronizer. You know: if a feature is not documented, it's not there :smile: .